composer.json

```diff
@@ -13,7 +13,8 @@
         "http-interop/http-factory-guzzle": "^1.2",
         "laravel/framework": "^12.1.1",
         "laravel/scout": "^10.13.1",
-        "meilisearch/meilisearch-php": "^1.13"
+        "meilisearch/meilisearch-php": "^1.13",
+        "sentry/sentry-laravel": "^4.13"
```

Comment:
After installing a new dep run `composer bump` this way we specify the actual version.